### PR TITLE
Add the possibility to separate odd updates vs failed updates

### PR DESCRIPTION
**Pull request by pypingou**
Odd updates are projects for which we find a new version that we find to
be inferior to the latest update we had.
Failed updates are plain failure to retrieve the newest update.

### CODE REVIEW

File: anitya/lib/model.py [hunk 1]
```diff
@@ -451,7 +451,15 @@ def updated(
             query = query.filter(
                 Project.logs != None,
                 Project.logs != 'Version retrieved correctly',
+                ~Project.logs.ilike('Something strange occured%'),
```
Comment (ralphbean) [inline anitya/lib/model.py]: Is this ``~`` intentional?  What does it do?

Comment (pypingou) [inline anitya/lib/model.py]: It's the negation, so in this case I do not want the project whose log start with 'Something strange occured'

Comment (ralphbean) [inline anitya/lib/model.py]: Cool!

**Comment by ralphbean** [comment]
:+1:

**Pull request merged**